### PR TITLE
Feature/delete label

### PR DESCRIPTION
**Pull request by dcangulo**
@kencruz07 
@ming-ancino 
@juliusb0205

### CODE REVIEW

File: app/controllers/labels_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class LabelsController < ApplicationController
+
+  def create
+    Label.create :name => params[:label][:name], :story_id => params[:label][:story_id]
+
+    respond_to do |format|
+      format.js {render inline: "location.reload();" }
+    end
```
Comment (juliusb0205) [inline app/controllers/labels_controller.rb]: Use reform

File: app/controllers/labels_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  def create
+    @label = LabelForm.new Label.new
+
+    if @label.validate params[:label]
+      @label.save
+    end
+
+    respond_to do |format|
+      format.js {render inline: "location.reload();" }
```
Comment (kencruz07) [inline app/controllers/labels_controller.rb]: Add space after `{`

File: app/controllers/labels_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    end
+  end
+
+  def destroy
+    label = Label.find params[:id]
+    label.destroy
+
+    respond_to do |format|
+      format.js {render inline: "location.reload();" }
```
Comment (kencruz07) [inline app/controllers/labels_controller.rb]: Same as above

File: app/views/projects/show.haml [hunk 4]
```diff
@@ -135,6 +137,23 @@
             %br
             %br
           %hr
+          .uk-text-bold Labels
+          .uk-grid-small
+          .uk-input
+            - vwork.labels.each do |label|
+              .uk-badge
+                = link_to project_label_path(@project, label), :method => :delete, :style => 'color: #fff;', :remote => true do
+                  %i{:'uk-icon' => 'close', :style => 'cursor: pointer;'}
```
Comment (kencruz07) [inline app/views/projects/show.haml]: When using string hash keys, omit leading `:` ex: `'uk-icon' => ...`

File: db/schema.rb [hunk 5]
```diff
@@ -10,6 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Comment (kencruz07) [inline db/schema.rb]: Fix merge conflicts

**Pull request merged**